pkg/apis/infinispan/v1/types_util.go

```diff
@@ -244,17 +245,29 @@ func (ispn *Infinispan) HasSites() bool {
 	return ispn.IsDataGrid() && ispn.Spec.Service.Sites != nil && len(ispn.Spec.Service.Sites.Locations) > 0
 }
 
-// GetSitesLocations returns remote site locations if localSiteName is provided
-// If localSiteName is not provided (empty string), all site locations will be returned
-func (ispn *Infinispan) GetSitesLocations(localSiteName string) (remoteLocations []InfinispanSiteLocationSpec) {
+// GetRemoteSiteLocations returns remote site locations
+func (ispn *Infinispan) GetRemoteSiteLocations() (remoteLocations []InfinispanSiteLocationSpec) {
 	for _, location := range ispn.Spec.Service.Sites.Locations {
-		if localSiteName != location.Name {
+		if ispn.Spec.Service.Sites.Local.Name != location.Name {
 			remoteLocations = append(remoteLocations, location)
 		}
 	}
 	return
```

Comment:
I guess you can sort the `remoteLocations` to be similar with `GetSiteLocationsName()`